lib/publiccloud/azure.pm

```diff
@@ -110,10 +111,13 @@ sub upload_img {
     }
 
     my ($img_name) = $file =~ /([^\/]+)$/;
-    $img_name =~ s/\.vhdfixed/.vhd/;
+    my $sku = (check_var('PUBLIC_CLOUD_AZURE_SKU', 'gen2')) ? 'V2' : 'V1';
```

Comment:
```suggestion
    my $sku = (check_var('PUBLIC_CLOUD_AZURE_SKU', 'gen1')) ? 'V1' : 'V2';
```

Reply:
Our workflow is currently based on gen1, it's probably a fuzz to change the default now.